instagram/README.md

```
@@ -0,0 +1,5 @@
+## Instagram
+
+This app is moved to independed repository.
+
+- https://github.com/instarchiver/instarchiver-backend
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)